src/main/scala/com/example/OKTable.scala

```diff
@@ -0,0 +1,37 @@
+package com.example
+
+import java.util.Properties
+
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.common.utils.Bytes
+import org.apache.kafka.common.serialization.Serdes
+import org.apache.kafka.streams.StreamsConfig
+import org.apache.kafka.streams.StreamsBuilder
+import org.apache.kafka.streams.KafkaStreams
+import org.apache.kafka.streams.kstream.Materialized
+import org.apache.kafka.streams.state.KeyValueStore
+import org.apache.kafka.streams.state.QueryableStoreTypes
+
+trait OKTable {
```

Comment:
as someone mentioned use `https://github.com/wishabi/datalib/blob/master/MicroserviceUtils/utils/src/main/scala/com/flipp/datalib/microservice/kafka/KafkaClientConfProvider.scala`

Reply:
I will definitely use this in the future, but will leave it out of my private repo for now.  Thanks for the heads up.